Dynatrace/dynatrace-operator
# Pass ActiveGate host as noProxy for CodeModules

## Description

Pass ActiveGate host as noProxy for CodeModules


## How can this be tested?

1.  Deploy dynakube with AG and with proxy
2. Deploy sample-app
3. make sure `ruxitagentproc.conf` has `noProxy` with value like: `xxx-activegate.dynatrace`.
4. make sure oneAget log has proper records:
```
root@php-glibc-all-inject-848c9c9f99-7n7l5:/var/www/html# cat /opt/dynatrace/oneagent-paas/log/php/ruxitagent_PHP-FPM_php-glibc-all-inject-__1.0.log
...
2024-01-09 10:04:09.121721 UTC [1/00000001] info    [native] Proxy ....................... http://squid.proxy.svc.cluster.local:3128/
2024-01-09 10:04:09.121729 UTC [1/00000001] info    [native] No-Proxy Hosts .............. xxx-activegate.dynatrace
...
2024-01-09 10:04:09.220309 UTC [10/0000000a] info    [comm  ] Connected to [https://xxxx-activegate.dynatrace:443/communication](https://xxx-activegate.dynatrace/communication)
```


## Checklist
